addon/components/basic-dropdown-content.ts

```diff
@@ -149,7 +149,8 @@ export default class BasicDropdownContent extends Component<Args> {
   @action
   animateOut(dropdownElement: Element): void {
     if (!this.animationEnabled) return;
-    let parentElement = dropdownElement.parentElement;
+    let parentElement =
+      dropdownElement.parentElement ?? this.destinationElement;
```

Comment:
this is causing a weird double animate out in my app

Reply:
Hello, thanks for the feedback @hoIIer, I can have a look, but could you please be more specific on your context, like:

- Browser + OS (Does the [animation on the docs](https://ember-basic-dropdown.com/docs/animations) behaves normally on your browser ?)
- Animation css you're using
- renderInPlace or not
- ...

Reply:
@hoIIer Also interested, It didn't cause any problem on mine.

Reply:
- happens in both chrome/firefox latest in osx
- using custom calculated position that sets css animation dynamically

```
animation: "popoutFadeInSlideLeft 0.1s ease-in-out"
right: 709.5
top: 433.1796875
```
```
  @keyframes popoutFadeInSlideLeft {
    from {
      opacity: 0;
      transform: translateX(-15px);
    }
    to {
      opacity: 1;
      transform: translateX(0px);
    }
  }
```
When I remove the `style.animation = ...` the popout looks normal

```
<BasicDropdown
  @calculatePosition={{this.calcPosition}}
  as |dd|
>
  {{yield (hash
    dd=dd
    trigger=(
      component 'popout-menu/trigger'
      componentStyle=(get-style this)
      disableMenu=@disableMenu
      dropdown=dd
      name=@name
      visible=@visible
    )
    menu=(
      component 'popout-menu/menu'
      componentStyle=(get-style this)
      dropdown=dd
      isDesktop=this.media.isDesktop
      onClickOverlay=this.onClickOverlay
      theme=@theme
    )
    componentStyle=(get-style this)
  )}}
</BasicDropdown>
```

Reply:
I extended the animation duration to 10 seconds and noticed when closing the dropdown, the content menu is immediately reset to it's starting position, but it still has the animation value set so it does a 2nd animation in before closing.

Reply:
Confirmed that in `4.0.2` the content menu does not reset to original position after clicking out to close the dropdown, the `<div id="ember-basic-dropdown-wormhole"></div>` just becomes empty again.

In `4.0.3` when clicking out to close the dropdown, the content menu appears again inside the parent container and with a `--clone` css class, leading to the 2nd animation before it's removed entirely.

Reply:
Can revisit later today, perhaps I need to adjust how I'm setting the animation? In this case I have to set it dynamically within the calcPosition function because my contextual dropdown is used in different contexts.